scratch/sim_correlator.py

```diff
@@ -38,10 +38,15 @@
 
 def parse_args(argv: Sequence[str] | None = None) -> argparse.Namespace:
     """Parse the command-line arguments (which may be specified as a parameter)."""
+
+    def _parse_input_labels(value: str) -> list[str]:
+        return value.split(",")
```

Comment:
I will admit, I was taking inspiration from our/katgpucbf's main.py parsing of e.g. `--wideband`. Happy to remove/update/tweak.

Reply:
I'm happy with that approach, but make this a top-level function rather than nesting it here (PEP 20: "Flat is better than nested.")